oci/oci.go

```diff
@@ -13,12 +13,11 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/containerd/cgroups"
 	"github.com/kubernetes-incubator/cri-o/utils"
 	rspec "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
-	"golang.org/x/sys/unix"
+	"istio.io/istio/pkg/log"
```

Comment:
We're laready using logrus in this package.  I can't compare the two packages, but using both at the same time seems overly complicated.

Reply:
OH NO! this is a goimports issue pulling in something from my GOPATH. Sorry

Reply:
fixed